lib/gollum-lib/committer.rb

```diff
@@ -53,8 +53,9 @@ def index
     # Returns a Gollum::Git::Actor.
     def actor
       @actor ||= begin
-        @options[:name]  = @wiki.default_committer_name if @options[:name].to_s.empty?
-        @options[:email] = @wiki.default_committer_email if @options[:email].to_s.empty?
+        name = @options[:name]
+        @options[:name]  = @wiki.default_committer_name if name.to_s.empty?
+        @options[:email] = @wiki.default_committer_email if name.to_s.empty?
```

Comment:
If `name == 'tom' isn't `name.to_s.empty?` false? I've tried to add this to the unit test from line 23.